### DISCUSSION TITLE
Help a newb add a path?

### DISCUSSION THREAD

**Discussion by loshlee** [ohmyzsh/ohmyzsh#10387] [Help] [answered]
Always a newb, because I never do more than dabble. I installed oh-my-zsh, but I need to add a path, I think, to fix a broken ruby gem, transcode-video. I tried adding my latest ruby gem path by inserting "# trying to make transcode-video work:
export PATH="/usr/local/opt/ruby/bin:/usr/local/Cellar/ruby/3.0.2_1/lib/ruby/gems/3.0.0/gems" at the end of oh-my-zsh.sh, but that was obviously wrong.  Should I add it to .zshrc? Some specific place in oh-my-zsh.sh? Somewhere else? Thanks.

**Comment by loshlee** [comment]
Still dabbling. Maybe this is the line I should add:
export PATH="/usr/local/opt/ruby/bin:/usr/local/lib/ruby/gems/2.4.0/gems:$PATH"

**Comment by loshlee** [comment]
The issue with transcode-video was solved a different way, but I'm still looking for a solution for adding oa single path item for oh-my-zsh. Or should newbies and Unix shell dabblers stay away from oh-my-zsh entirely?

**Comment by mcornella** [comment, marked as answer]
We generally recommend adding a PATH in your zshrc file, preferably at the top (see [example in the template zshrc file](https://github.com/ohmyzsh/ohmyzsh/blob/0520c2e30934a6e01b27988dca5bbbe3511d6868/templates/zshrc.zsh-template#L1-L2)).

So if you want to add a PATH, add at the top of your zshrc file, either

1. A directory with more precedence:
   ```zsh
   export PATH="/path/to/directory:$PATH"
   ```
2. A directory with less precedence:
   ```zsh
   export PATH="$PATH:/path/to/directory"
   ```

Double quotes important. The rest of the problem is figuring out what directory to add to `$PATH`.

**Reply by mcornella** [reply]
NOTE: it doesn't matter whether you're using Oh My Zsh or not. Adding a directory to `$PATH` is always the same, even if you're using bash or zsh.

**Reply by loshlee** [reply]
Thank you , mcornella.
